recipe-server/normandy/settings.py

```diff
@@ -389,6 +399,13 @@ class Production(Base):
     DEFAULT_FILE_STORAGE = values.Value('normandy.base.storage.S3Boto3PermissiveStorage')
     AWS_S3_FILE_OVERWRITE = False
 
+    # Custom CORS settings that overrides the CORS class's configuration.
+    # In production we harden it down a bit extra.
+    CORS_ORIGIN_ALLOW_ALL = values.BooleanValue(False)
+    CORS_ORIGIN_WHITELIST = values.ListValue([
+        'https://normandy-admin.prod.mozaws.net'
+    ])
+
 
 class ProductionReadOnly(Production):
```

Comment:
I would prefer if the `ReadOnly` variants of the configs set `CORS_ALLOW_METHODS = ['GET', 'OPTIONS']` (ie, only non-writing, idempotent methods). This is not strictly necessary, but it further hardens the API, and it presents more accurate information in `Access-Control-Allow-Methods`.

Generally, the read only servers don't have write access to the DBs, and the APIs don't allow any write methods. This is mostly a reflection of that current policy.